backend/django/core/views/api_admin.py

```diff
@@ -40,18 +40,34 @@ def label_distribution(request, project_pk):
     users.append(project.creator)
     users.extend([perm.profile for perm in project.projectpermissions_set.all()])
 
+    def sortCount(label):
+        label_count = 0
+        for u in users:
+            label_count += DataLabel.objects.filter(profile=u, label=label).count()
+        return label_count
+
+    # Sort labels by the count
+    labels.sort(key=sortCount, reverse=True)
+
     dataset = []
     all_counts = []
```

Comment:
It doesn't look like all_counts is used for anything anymore. I think we can just remove it. 

We might want to think about what should be returned if there is no data. Right now if we look at the admin chart for the "no labels" project, we see this:

<img width="1113" alt="Screen Shot 2021-07-29 at 2 03 22 PM" src="https://user-images.githubusercontent.com/10951569/127542746-df2b2a17-6fd5-4dc5-b3fe-493881e06db8.png">

I don't hate this, but we could alternatively show a message like we do in the IRR chart.

Reply:
I modified the backend some to only return data when counts > 0, and that solved the issue with the chart rendering but having no data. It now looks like the IRR chart when there is no data